.docker/docker-config.js

```diff
@@ -0,0 +1,81 @@
+const roles = require('./config.js');
+
+const config = {
+  // Bot Owner, level 10 by default. You no longer need to supply the owner ID, as the bot
+  // will pull this information directly from it's application page.
+  
+  // Your Bot's Token. Available on https://discordapp.com/developers/applications/me
+  'token': '{{token}}',
+
+  // Default per-server settings. These settings are entered in a database on first load, 
+  // And are then completely ignored from this file. To modify default settings, use the `conf` command.
+  // DO NOT REMOVE THIS BEFORE YOUR BOT IS LOADED AND FUNCTIONAL.
+
+  'defaultSettings' : {
```

Comment:
Slightly redundant to have this info in util/ setup files and in here.

Reply:
Novuh's settings system is pretty confusing, but it works. I say we keep the stuff in there so it doesn't break again, like when I tried lol. Or you could rewrite his settings system 🤷‍♀️